MANIFEST.in

```diff
@@ -1,6 +1,9 @@
-# Source distribution files.
-include README.md
-include setup.py
+include README.rst LICENSE tox.ini .travis.yml docs/Makefile manage.py
```

Comment:
* `tox.ini`: we're not using tox?
* Shouldn't include `.travis.yml` I don't think
* I would include `setup.cfg` and (I think) the requirements files.
* There doesn't seem to be a `tests` directory (yet?)
* I've seen some projects just do `graft docs` to add the docs